semgrep/semgrep/evaluation.py

```diff
@@ -1,97 +1,16 @@
-import copy
```

Comment:
Deleted boolean composition logic from this file now that it is handled in semgrep-core but kept the string interpolation logic